server/models/Session.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * A object to indicate current session, currently, use for database session, like a transaction session. So with knowing the current database transaction session, we can easily break the current process, and rollback the operation, and if encounter any accidentally problem or failure, we can rollback all operations.
+ */
+const knex = require("../database/knex");
+
+class Session{
```

Comment:
Great.  Seems like this object could also be added to the microservices template

Reply:
Created pull request #9 to treetracker-microservice-template with this addition. [Here is the link.](https://github.com/Greenstand/treetracker-microservice-template/pull/9)